vet.sh

```diff
@@ -120,11 +114,26 @@ if [[ -z "${VET_SKIP_PROTO}" ]]; then
     (git status; git --no-pager diff; exit 1)
 fi
 
-# - Check that our modules are tidy.
+# - gofmt, goimports, golint (with exceptions for generated code), go vet,
+# go mod tidy.
+# Perform these checks on each module inside gRPC.
 if go help mod >& /dev/null; then
```

Comment:
We no longer need this, since we only support Go versions with `mod` support; please remove it, thanks!